voidcrew/code/controllers/subsystem/job.dm

```diff
@@ -0,0 +1,15 @@
+/datum/controller/subsystem/job/proc/get_manifest()
```

Comment:
leave the old get_manifest then just make this overwrite it to tg rebase-proof it

Reply:
`get_manifest` is part of `datacore` on tg, this is shiptest custom